contributing.md

```diff
@@ -0,0 +1,39 @@
+# Contributing guidelines
+
+If you're willing to create a new PR on Starport, make sure that you read and comply with this document.
+
+Start a new [Discussion](https://github.com/tendermint/starport/discussions/new) if you want to propose changes to this document.
```

Comment:
@ilgooz we want users to start a discussion for changes to this contributing.md file? or for changes to the contributing process?

Reply:
To the md file, we don't necessarily need them to start a discussion but if they think something needs to be different or another thing needs to be added, they're welcome.